service/src/main/java/org/jboss/sbomer/service/feature/sbom/pyxis/PyxisValidatingClient.java

```diff
@@ -71,6 +74,11 @@ public PyxisRepositoryDetails getRepositoriesDetails(
         Set<ConstraintViolation<PyxisRepositoryDetails>> violations = validator.validate(prd);
         if (violations.isEmpty())
             return prd;
+        log.warn(
+                "The deserialized Pyxis repositry for nvr: {} and includes:{} violated the following constraints:{}",
```

Comment:
Nitpicking: a space before second and third  `{}`.

Reply:
This would also annoy me when its time to grep through logs :grin: